crates/ide_completion/src/context.rs

```diff
@@ -940,6 +940,7 @@ const OP_TRAIT_LANG_NAMES: &[&str] = &[
     "deref",
     "div_assign",
     "div",
+    "eq",
```

Comment:
fun fact, the lang item for `PartialOrd` is `partial_ord`, but for `PartialEq` its `eq`